Sources/VitaminCore/Components/Chip/VitaminChipVariant.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  Vitamin iOS
+//  Apache License 2.0
+//
+
+import UIKit
+
+/// Enum that holds info about a var
+public enum VitaminChipVariant {
```

Comment:
Ok. I hesitated, and thought it could be useful in the future (a lot of things we initially put in UIKit was moved to Core).
But no problem to move it to UIKit, and we'll move only what is really useful when this will be implemented on SwiftUI side

Reply:
done